docs/kit-team/README.md

```diff
@@ -0,0 +1,100 @@
+# Kit Team
```

Comment:
RFC: I'm personally wary about renaming this section to "Kit Team", over simply "Kit". It feels strange to group everything under the team, rather than the domain, even though ~100% of what's considered "Kit" falls under the remit of the kit team.

In much the same way, renaming the "Competition" section "Competition Team" would also feel weird, if not more so, as almost everything else falls under that domain.

Personally, I'd have 2 tabs. 1 for the kit itself (as-is), 1 for the kit team organisations. But that's me.

(Intentionally not done as a review so as not to block merging, just registering concern. More than happy to be proved wrong!)

Reply:
Why not put this doc in the "Volunteering" section and leave the "Kit" section as it was?

Reply:
My feeling towards this is the current runbook does not have the information about how either the compeition cycle ran last year, or how the kit has been managed the last couple of years. As such modifying the runbook to describe the new stucture of SR seems logical. 

The current content of the Kit Tab will likly all be written from scratch, as very little of it is now relevent to how the kit will be orginised intthe future. With the Assets team taking over the orginisation of the kit, and the develeopment being run by the relevent project team.

(Also on a side note, kit isnt really a good name for the current content either, as it describes the duties of a group of people whos purpose is to look after the kit, it doesnt talk much about the kit itself, which is the purpose of the docs)

Reply:
One thing to note is that having two levels of the hierarchy labelled as "team" makes it hard to understand how teams are grouped.
Labelling the project teams as sub-teams makes it clear they are grouped under the kit team rather than separate top level teams.

Reply:
> My feeling towards this is the current runbook does not have the information about how either the compeition cycle ran last year, or how the kit has been managed the last couple of years. As such modifying the runbook to describe the new stucture of SR seems logical.

For SR2019 the handling of the kit was pretty close to what is currently documented, at least as far as kit collation and logistics go. I don't recall specifically referring to the packing docs, though I think we did something very similar to what is documented. For sure there are things we missed or did slightly differently (as is natural), but the intent was to follow the runbook's guidance. (It also didn't help that we were in the process of pulling the runbook together at the time)

> The current content of the Kit Tab will likly all be written from scratch, as very little of it is now relevent to how the kit will be orginised intthe future. With the Assets team taking over the orginisation of the kit, and the develeopment being run by the relevent project team.

This seems reasonable, though I'm not sure why this has any relation to the renaming of the section?

> (Also on a side note, kit isnt really a good name for the current content either, as it describes the duties of a group of people whos purpose is to look after the kit, it doesnt talk much about the kit itself, which is the purpose of the docs)

The section describes the general topic area within SR. I've outlined in [my comment](https://github.com/srobo/runbook/pull/73#issuecomment-578409259) a bit more about this, but the idea is that the runbook should document things not people†. It's somewhat unfortunate that it's currently framed around people, however this is due to a lack of time following the unification of the previous disparate docs.

† this gets a little complicated when consider the volunteering section, which is obviously also about people, but probably best thought about as "volunteering" being the _thing_ which is being documented. When viewed like that, the ways in which you can volunteer are therefore natural members of that topic.